src/main/host/syscall/clone.h

```diff
@@ -0,0 +1,9 @@
+/*
+ * The Shadow Simulator
+ * See LICENSE for licensing information
+ */
+
+#include "main/host/syscall_handler.h"
+#include "main/host/syscall_types.h"
+
+SysCallReturn syscallhandler_clone(SysCallHandler* sys, const SysCallArgs* args);
```

Comment:
Do you want to be consistent with the other handlers and add an include guard and use the `SYSCALL_HANDLER` define? E.g.:

```c
#ifndef SRC_MAIN_HOST_SYSCALL_CLONE_H_
#define SRC_MAIN_HOST_SYSCALL_CLONE_H_

#include "main/host/syscall/protected.h"

SYSCALL_HANDLER(clone);

#endif /* SRC_MAIN_HOST_SYSCALL_CLONE_H_ */
```